packages/expo-dev-launcher/bundle/screens/ExtensionsScreen.tsx

```diff
@@ -1,173 +1,230 @@
+import { StackNavigationProp } from '@react-navigation/stack';
 import {
-  Heading,
   Spacer,
   View,
   Text,
   Button,
-  useExpoPalette,
   Row,
   ChevronRightIcon,
-  Divider,
-  BranchIcon,
-  UpdateIcon,
   ExtensionsIcon,
+  scale,
+  Divider,
+  Heading,
 } from 'expo-dev-client-components';
 import * as React from 'react';
-import { SafeAreaView } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
 
-export function ExtensionsScreen() {
-  const extensions = [];
-  const palette = useExpoPalette();
+import { ActivityIndicator } from '../components/ActivityIndicator';
+import { AppHeader } from '../components/AppHeader';
+import { EASBranchRow } from '../components/EASUpdatesRows';
+import { EmptyBranchesMessage } from '../components/EmptyBranchesMessage';
+import { useBuildInfo } from '../providers/BuildInfoProvider';
+import { useUser, useUserActions } from '../providers/UserContextProvider';
+import { useBranchesForApp } from '../queries/useBranchesForApp';
+import { ExtensionsStackParamList } from './ExtensionsStack';
 
-  return (
-    <SafeAreaView style={{ flex: 1 }}>
-      <View flex="1">
-        {extensions.length === 0 && (
-          <View bg="default" mx="medium" py="medium" px="medium" rounded="medium">
-            <View align="centered">
-              <ExtensionsIcon />
-            </View>
-            <Spacer.Vertical size="medium" />
-            <View px="small">
-              <Text size="small" align="center">
-                Extensions allow you to customize your development app with additional capabilities.
-              </Text>
-            </View>
+const extensions = ['updates'];
+const hasEASUpdatesInstalled = extensions.includes('updates');
```

Comment:
this logic has been updated somewhat since your review as well - sorry I should have commented about having this be just a placeholder until I got to this commit: 5a3dec59b3916ab4695455abac901c88d061d82c